data/data/aws/bootstrap/vpc/master-elb.tf

```diff
@@ -0,0 +1,74 @@
+data "aws_lb" "api_internal" {
```

Comment:
Do we really need all of these `data` objects? All I see being used from the vpc module are the target group ARNs, the VPC ID, and the master security group ID.

Reply:
I've removed  many of the data sources. There still may be some that are unused.

Reply:
It seems that anything more removed causes the install to fail. I think everything here is required.